packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -38,6 +39,26 @@ export interface RadioTokens extends FontTokens, IColorTokens, IForegroundColorT
    */
   radioBorderWidth?: number;
 
+  /**
+   * Variant of label subtext
+   */
+  subtextVariant?: keyof Variant;
+
+  /**
+   * Padding between label and label subtext
+   */
+  marginTop?: ViewStyle['marginTop'];
```

Comment:
Should the names of these signify they're specific towards label subtext?

Ie. `subtextMarginTop`

Right now, I would assume that these values are for the label itself

Reply:
Thanks for the suggestion! I agree that these names should be more descriptive.